packages/atomic/cypress/integration/smart-snippet-suggestions-assertions.ts

```diff
@@ -0,0 +1,30 @@
+import {AnalyticsTracker} from '../utils/analyticsUtils';
+import {should} from './common-assertions';
+import {SmartSnippetSuggestionsSelectors} from './smart-snippet-suggestions-selectors';
+
+export function assertAnswerTopMargin(margin: number) {
+  it(`has a ${margin}px gap between the first expanded button and the snippet`, () => {
+    SmartSnippetSuggestionsSelectors.questionExpandedButton()
+      .distanceTo(SmartSnippetSuggestionsSelectors.answer)
+      .should('have.property', 'vertical', margin);
+  });
+}
+
+export function assertAnswerBottomMargin(margin: number) {
+  it(`has a ${margin}px gap between the snippet and the footer`, () => {
+    SmartSnippetSuggestionsSelectors.answer()
+      .distanceTo(SmartSnippetSuggestionsSelectors.footer)
+      .should('have.property', 'vertical', margin);
+  });
+}
+
+export function assertlogOpenSmartSnippetSuggestionsSource(log: boolean) {
+  it.skip(`${should(log)} log a openSmartSnippetSource click event`, () => {
```

Comment:
did you mean to leave this skipped?

Reply:
yes. Unfortunately, it doesn't log the correct click event yet.